src/main/java/ua/com/conductor/model/User.java

```diff
@@ -0,0 +1,62 @@
+package ua.com.conductor.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "user")
```

Comment:
It's better to use plural for table names. Moreover, such name for the table may conflict with internal MySQL table for users